hadoop-hdds/server-scm/src/test/java/org/apache/hadoop/hdds/scm/container/balancer/MockedSCM.java

```diff
@@ -137,7 +137,6 @@ public String toString() {
   }
 
   public @Nonnull ContainerBalancerTask startBalancerTask(@Nonnull ContainerBalancerConfiguration config) {
-    init(config);
```

Comment:
In TestContainerBalancerDatanodeNodeLimit, each time init(config) is called, it is followed by startBalancerTask. We could retain this line and avoid calling mockedSCM.init() in every test case in TestContainerBalancerDatanodeNodeLimit.